Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/mail.yml

```diff
@@ -493,6 +493,16 @@
     contents:
     - id: CombatKnife
 
+- type: entity
+  noSpawn: true
+  parent: BaseMail
+  id: Mailiceaxe
+  suffix: Knife
```

Comment:
what should i change the suffix to then?

Reply:
Suffixes appear only to mappers and are for giving them useful information, usually in the event of two entities having the same name. Probably not useful here at all since this is a nospawn item therefore does not appear in the entity spawn panel.

Reply:
i think i know what your talking about

Reply:
fixed it